browser.js

```diff
@@ -193,9 +193,7 @@ define(function(require, exports, module) {
                 
                 var src = getIframeSrc(iframe);
                 var path = calcRootedPath(cleanIframeSrc(src));
-                
-                tab.title = 
-                tab.tooltip = "[B] " + path;
```

Comment:
Ah, sorry for that! Added back for now, and commented so that it's clear. Thanks!